### DISCUSSION TITLE
Nonexistent class imported and used in code example [Documentation]

### DISCUSSION THREAD

**Discussion by AM-ops** [sqlalchemy/sqlalchemy#9305] [Usage Questions] [answered]
Breadcrumbs to location of code:
[Class Mapping API](https://docs.sqlalchemy.org/en/20/orm/mapping_api.html) -> [DeclarativeBase](https://docs.sqlalchemy.org/en/20/orm/mapping_api.html#sqlalchemy.orm.DeclarativeBase) (The Annotated Example)

Code given as follows:
```python
from typing import Annotation

from sqlalchemy import BigInteger
from sqlalchemy import MetaData
from sqlalchemy import String
from sqlalchemy.orm import DeclarativeBase

bigint = Annotation(int, "bigint")
my_metadata = MetaData()

class Base(DeclarativeBase):
    metadata = my_metadata
    type_annotation_map = {
        str: String().with_variant(String(255), "mysql", "mariadb"),
        bigint: BigInteger()
    }
```

The class in question is `Annotation` 

No version of Python from 3.7 till 3.11 has such a class in the `typing` library. 

The 3.11 version of the library can be found here: [https://github.com/python/cpython/blob/3.11/Lib/typing.py](https://github.com/python/cpython/blob/3.11/Lib/typing.py)

**Comment by zzzeek** [comment, marked as answer]
fixed in d50315cbd6b448287b18, this refers to ``typing.Annotated``.  A complete example is at https://docs.sqlalchemy.org/en/20/orm/declarative_tables.html#mapping-multiple-type-configurations-to-python-types

**Comment by AM-ops** [comment]
Thanks! :)

**Comment by michael-newsrx** [comment]
This also seems to be an issue here: [Quickstart](https://docs.sqlalchemy.org/en/20/orm/quickstart.html)

**Reply by zzzeek** [reply]
> This also seems to be an issue here: [Quickstart](https://docs.sqlalchemy.org/en/20/orm/quickstart.html)

Hi I dont see the phrase "Annotation" or "Annotated" in this document, and the document does not discuss this concept. The document is also a doctest that's part of our CI so all code there is fully valid and runnable.  Can you be more specific towards the issue you are referring?

**Reply by michael-newsrx** [reply]
Yep, it was a PyCharm cache issue. It was showing some things from the orm package but not others... so I was really confused.
